Mathlib/Algebra/Module/FinitePresentation.lean

```diff
@@ -73,22 +73,22 @@ end Semiring
 
 section Ring
 
-variable (R M N) [Ring R] [AddCommGroup M] [Module R M] [AddCommGroup N] [Module R N]
-
-universe u in
-variable (R M : Type u) [Ring R] [AddCommGroup M] [Module R M] in
+universe u v in
+variable (R : Type u) (M : Type v) [Ring R] [AddCommGroup M] [Module R M] in
 /-- A finitely presented module is isomorphic to the quotient of a finite free module by a finitely
 generated submodule. -/
 theorem Module.FinitePresentation.equiv_quotient [fp : Module.FinitePresentation R M] :
-    ∃ (L : Type u) (_ : AddCommGroup L) (_ : Module R L) (K : Submodule R L) (_ : M ≃ₗ[R] L ⧸ K),
-      Module.Free R L ∧ Module.Finite R L ∧ K.FG := by
+    ∃ (L : Type max u v) (_ : AddCommGroup L) (_ : Module R L) (K : Submodule R L)
```

Comment:
It's possible to het Type u here even though M is Type v; that version is stronger and potentially more useful.

Reply:
Do you want to include that in this PR, or leave it as follow up?

Reply:
Let me change it to use `Small` instead.